src/system/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,48 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Footer } from './Footer';
+
+const links = [
+	{
+		children: 'Link1',
+		href: 'https://wpvip.com/',
+	},
+	{
+		children: 'Link2',
+		href: 'https://docs.wpvip.com/',
+	},
+];
+
+describe( '<LinkExternal />', () => {
```

Comment:
Maybe a typo? `<Footer />`

Reply:
Definitely! I got mixed up with the test name which I'll comment on below